### PR TITLE
Add cookie domain, redirects and fix some issues

### PR DESCRIPTION
**Pull request by pjarzyniak**
# PR Details

Fix two issues to start my own project based on modseven

### Description

1. Cookie domain is in my configuration required, so I add this in config file and set in index.php
2. Fix: class Session have error - not running on PHP 7.4.12, because returns null where return type is string.

### Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ x ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.
- [ x ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by pjarzyniak** [comment]
I have added more fixes, missing http redirects and so on. Finally the project works and can be used.

**Comment by toitzi** [comment]
Thanks a lot!

File: system/config/app.php [hunk 1]
```diff
@@ -15,6 +15,7 @@
         'salt' => FALSE,
         'httponly' => FALSE,
         'secure' => FALSE,
+        'domain' => NULL,
```
Comment (UchihaSV) [inline system/config/app.php:18]: need change cookie domain NULL to some string, bacause it's give error https://github.com/modseven/core/issues/3

Comment (pjarzyniak) [reply system/config/app.php:18]: Yes, you have to set cookie domain in your project before start

**Pull request merged**